security/pkg/pki/util/keycertbundle_test.go

```diff
@@ -57,12 +57,12 @@ func TestKeyCertBundleWithRootCertFromFile(t *testing.T) {
 	for id, tc := range testCases {
 		bundle, err := NewKeyCertBundleWithRootCertFromFile(tc.rootCertFile)
 		if err != nil {
-			if len(tc.expectedErr) == 0 {
+			if tc.expectedErr == "" {
```

Comment:
Sorry, I thought the original check is equivalent to the new one, no?

Reply:
they are. I believe str == "" is the canonical:
https://golang.org/pkg/strings/?m=all#Compare:
```
Compare returns an integer comparing two strings lexicographically. 
The result will be 0 if a==b, -1 if a < b, and +1 if a > b.

Compare is included only for symmetry with package bytes. 
It is usually clearer and always faster to use the built-in string
comparison operators ==, <, >, and so on. 
```

Reply:
@mydpt since the two checks are equivalent, I can certainly revert and keep the existing.
WDYT?

Reply:
Sorry for the late reply. You can keep your change :)